tests/js/ttsHookupTests.js

```diff
@@ -13,17 +13,109 @@ https://github.com/gpii/universal/LICENSE.txt
 
     fluid.registerNamespace("gpii.tests");
 
-    jqUnit.test("gpii.firstDiscovery.tts.fdHookup.bindKeypress", function () {
-        jqUnit.expect(1);
-        var elm = $(".bindKeypress-test");
-        var keyCode = 104; // 'h'
+    fluid.defaults("gpii.tests.mock.firstDiscoveryEditor", {
+        gradeNames: ["fluid.viewRelayComponent", "autoInit"],
+        selectors: {
+            panel: ".gpiic-fd-prefsEditor-panel"
+        },
+        model: {
+            currentPanelNum: 1
+        },
+        components: {
+            selfVoicing: {
+                type: "fluid.standardComponent",
+                options: {
+                    gradeNames: ["gpii.firstDiscovery.msgLookup"],
+                    model: {
+                        enabled: true
+                    },
+                    messageBase: {
+                        "panelMsg": "This is step %currentPanel of %numPanels. %instructions Press 'h' for help."
+                    },
+                    invokers: {
+                        queueSpeech: "{firstDiscoveryEditor}.events.onTestQueueSpeech.fire"
+                    }
+                }
+            }
+        },
+        events: {
+            onTestQueueSpeech: null
+        },
+        listeners: {
+            "onCreate.setPanels": {
+                listener: "fluid.set",
+                args: ["{that}", "panels", "{that}.dom.panel"],
+                priority: "first"
+            }
+        }
+    });
+
+    fluid.defaults("gpii.tests.firstDiscovery.tts.fdHookup", {
+        gradeNames: ["gpii.tests.mock.firstDiscoveryEditor", "gpii.firstDiscovery.tts.fdHookup", "autoInit"]
+    });
+
+    fluid.defaults("gpii.tests.fdHookupTests", {
+        gradeNames: ["fluid.test.testEnvironment", "autoInit"],
+        components: {
+            fdHookup: {
+                type: "gpii.tests.firstDiscovery.tts.fdHookup",
+                container: ".gpiic-tests-firstDiscovery-tts-fdHookup"
+            },
+            ttsHookupTester: {
+                type: "gpii.tests.ttsHookupTester"
+            }
+        }
+    });
+
+    fluid.defaults("gpii.tests.ttsHookupTester", {
+        gradeNames: ["fluid.test.testCaseHolder", "autoInit"],
+        modules: [{
+            name: "Tests the fdHookup component",
+            tests: [{
+                expect: 1,
+                name: "gpii.firstDiscovery.tts.fdHookup.getCurrentPanelInstructions",
```

Comment:
You probably wanna use a meaningful message such as "Test gpii.firstDiscovery.tts.fdHookup.getCurrentPanelInstructions()"